### PR TITLE
combine installation+supported os pages

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes https://github.com/meilisearch/documentation/issues/2148

### CODE REVIEW

File: learn/getting_started/installation.mdx [hunk 1]
```diff
@@ -148,3 +150,27 @@ To deploy Meilisearch on a cloud service, follow one of our dedicated guides:
 - [Koyeb](/learn/cookbooks/koyeb.md)
 - [Qovery](/learn/cookbooks/qovery.md)
 - [Railway](/learn/cookbooks/railway.md)
+
+## Supported operating systems
+
+This section lists operating systems Meilisearch officially supports and tests with every new release. Meilisearch binaries might still run in unsupported environments. Refer to the [changelog](https://github.com/meilisearch/MeiliSearch/releases) for more information on operating system support changes.
```
Comment (dichotommy) [inline learn/getting_started/installation.mdx:156]: To me it does not seem helpful to point people to the changelog for information about operating system support, as any relevant info will be buried among a lot of other stuff. This page exists to collect that information so I don't see the value in sending people elsewhere.

```suggestion
This section lists operating systems Meilisearch officially supports and tests with every new release. Meilisearch binaries might still run in unsupported environments. 
```

File: learn/getting_started/installation.mdx [hunk 2]
```diff
@@ -148,3 +150,27 @@ To deploy Meilisearch on a cloud service, follow one of our dedicated guides:
...
+
+If the provided [binaries](/learn/getting_started/installation.md#local-installation) don't work for you, try building Meilisearch from source. If compilation fails, Meilisearch is not compatible with your machine.
```
Comment (dichotommy) [inline learn/getting_started/installation.mdx:158]:
```suggestion
If the provided [binaries](/learn/getting_started/installation.md#local-installation) do not work on your operating system, try building Meilisearch [from source](#local-installation). If compilation fails, Meilisearch is not compatible with your machine.
```

File: learn/getting_started/installation.mdx [hunk 3]
```diff
@@ -1,8 +1,10 @@
-# Installation
+# Installation and supported operating systems
```
Comment (dichotommy) [inline learn/getting_started/installation.mdx:1]:
```suggestion
# Installation and operating systems
```

**Review by dichotommy — CHANGES_REQUESTED** [review]

**Comment by maryamsulemani97** [comment]
closing for #2279

**Pull request closed without merging**